src/m365/aad/commands/o365group/o365group-recyclebinitem-restore.ts

```diff
@@ -37,9 +37,9 @@ class AadO365GroupRestoreCommand extends GraphCommand {
     }
 
     const requestOptions: any = {
-      url: `${this.resource}/v1.0/directory/deleteditems/${args.options.id}/restore/`,
+      url: `${this.resource}/v1.0/directory/deleteditems/${args.options.id}/restore`,
       headers: {
-        'accept': 'application/json;odata.metadata=none'
```

Comment:
Is there a reason why this had to be changed?

Reply:
Yes. The Graph API gets success after adding "Content-type":"application/json" to the Request header
For reference: https://docs.microsoft.com/en-us/graph/api/directory-deleteditems-restore?view=graph-rest-1.0&tabs=http

Please check.

Reply:
I see it indeed being referenced, but since there is no body I think that's why we didn't include it initially. We can add it. I'd keep the accept header because it allows us to avoid retrieving metadata which we don't need.